resources/assets/uad_lists.json

```
@@ -2690,6 +2690,15 @@
     "labels": [],
     "removal": "Recommended"
   },
+  {
+    "id": "com.motorola.discovery",
+    "list": "Oem",
+    "description": "Help you find your lost phone. An excuse to track your location.",
```

```suggestion
    "description": "Helps you find your lost phone. Might also be an excuse to track your location.",
```
You're making an assumption here without being certain, so I changed the sentence.

---

Approved!